src/cpp/server/DBActiveSessionStorage.cpp

```diff
@@ -0,0 +1,417 @@
+/*
+ * DBActiveSessionStorage.cpp
```

Comment:
Should we have a sub-directory called "sessions" for server specific session feature code?  It could hold all of the DBActiveSession* features.

Reply:
That's a good idea